bindata/bootkube/bootstrap-manifests/etcd-member-pod.yaml

```diff
@@ -8,7 +8,7 @@ metadata:
 spec:
   containers:
   - name: etcd
-    image: {{ .Images.Etcd }}
+    image: {{ .Image }}
```

Comment:
would we ever need to include the operator image or some other image? I suppose we can change that in the future if needed.

Reply:
Yeah, I'm partial to YAGNI.